create.ts

```diff
@@ -0,0 +1,804 @@
+import question from "https://raw.githubusercontent.com/ocpu/question-deno/master/mod.ts";
```

Comment:
This is an important feature that Ultra needs to have. But I have some problems with the `create.ts` file in terms of its refactorability and testability. Here a few of my thoughts about how to refactor this code:

The code is too dense. It should be broken down into individual functions and other functions moved to files in `lib/create`. 

- Group similar functions in the same file (e.g. `Questions.ts`, `FileTasks.ts`, `Content.ts`).

- In `create.ts`, just have a `create()` function that calls the other functions. Maybe `createUltraApp()` could be in this file, but create() should call it.

- You should move all the TS types into a `types.ts` file. 

- The question/answer sections should move to functions that return either a Promise<boolean> (for confirm type) or Promise<string> (for list type). 

- Functions that help populate the config object should return the config obj.

- Functions that use the config object should take it as an argument.

I hope this comment does not discourage you, Benton. You have created a great new feature, and I hope we see more contributions from you.

Reply:
Reach out to me on Discord if you need some help

Reply:
Hi @cdoremus 

I was under the impression Omar wanted me to PR this now in order to start using it while it gets refactored. I basically just extended `init.ts` in order to add the additional prompts to get a working prototype. We might end up going a different route re. multiple choice as to not need the --unstable flag - in any case, this is not intended to be production ready.

Reply:
If you have any ideas on a nicer way to handle conditionals in files besides template literals let me know! That part is particularly clunky.

Reply:
> Hi @cdoremus
> 
> I was under the impression Omar wanted me to PR this now in order to start using it while it gets refactored. I basically just extended `init.ts` in order to add the additional prompts to get a working prototype. We might end up going a different route re. multiple choice as to not need the --unstable flag - in any case, this is not intended to be production ready. If you want I can close this and re-open once I finish refactoring it.

It's up to Omar how we want to go with this, but my experience is that it's easier to refactor now than later.

Reply:
Yes, I think it's good to have this as a jumping off point -- we can work on refactoring this a bit more, but as far as functionality goes this ticks a lot of boxes for people starting out.